### ISSUE TITLE
 Why multiply the heatmap by 255?

### ISSUE THREAD

**Issue by f6ra07nk14** [facebookresearch/InterHand2.6M#38]
When I trace the training code in [model.py](https://github.com/facebookresearch/InterHand2.6M/blob/75a2d29421ad9dd8607e7948dcfbccf706b1eef0/main/model.py#L38) , I saw that "heatmap" is multiplied by 255 on the 38 line.
Could this value be an arbitrary scale value, or have other physical meanings?

Thanks in advance.

**Comment by mks0601** [comment]
No it doesn't have special meanings. Just to scale the loss values. I think there would be marginal performance difference if you do not multiply 255.
